.github/workflows/bump-version.yaml

```
@@ -63,6 +63,14 @@ jobs:
         id: commit
         uses: stefanzweifel/git-auto-commit-action@v5
         with:
+          # We need to commit as a user to ensure CI is run on the commit
+          # We need to push as an admin/owner to bypass branch-protection
+          token: ${{ secrets.HASHALITE_PAT }}
+          # Ensure we commit as a bot, since we can't reliably get an author slug from `github.actor`
+          commit_user_name: github-actions[bot]
+          commit_user_email: 41898282+github-actions[bot]@users.noreply.github.com
+          commit_author: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
```

```suggestion
          commit_author: ${{ github.actor }} <41898282+github-actions[bot]@users.noreply.github.com>
```

This would attribute the author's name to the github _username_, which could be a decent compromise that doesn't involve looking up the `/users/<username>` endpoint?

It'd still use the bot email though, and it wouldn't use "display" names.

e.g. `--author "hashalite <41898282+github-actions[bot]@users.noreply.github.com>"`

---

Yes, this would be better